android/UtilsPlugin.java

```diff
@@ -241,6 +259,39 @@ public void onNewIntent(Intent gameIntent) {
 		}
 	}
 
+	public void openAppSettings() {
+		Uri packageURI = Uri.parse("package:" + _context.getPackageName());
+		Intent intent = new Intent("android.settings.APPLICATION_DETAILS_SETTINGS", packageURI);
+		intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+		_context.startActivity(intent);
+	}
+
+	public void showEnableNotificationPopup(String params) {
+		try {
+			_activity.runOnUiThread(new Runnable() {
+				public void run() {
+					new AlertDialog.Builder(_activity)
+					.setTitle("Enable Notifications")
+					.setMessage("To receive Sudoku of the Day everyday!")
```

Comment:
this shouldnt be hard coded. What if we need same functionality for some other notification?